proto/v1/structure/languages.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "proto/v1/structure/versions.proto";
+
+option java_package = "org.oppia.proto.v1.structure";
+option java_multiple_files = true;
+
+// Structure corresponds to LanguageProtosVersion.
+enum LanguageCode {
+  LANGUAGE_CODE_UNSPECIFIED = 0;
+  ENGLISH = 1;
+  ARABIC = 2;
+  HINDI = 3;
+  HINGLISH = 4;
+  BRAZILIAN_PORTUGUESE = 5;
+}
+
+
+// Strucutre of the single subtitled html. 
+message SubtitledHtml {
+  LanguageProtosVersion structure_version = 1;
+
+  // Id of the content whomsoever corresponds to SubtitledHtml.
+  string content_id = 2;
+
+  // Html format data of the content whomsoever corresponds to SubtitledHtml.
+  string html = 3;
+}
+
+// Strucutre of the single recorded voiceover. 
+message RecordedVoiceovers {
+  LanguageProtosVersion structure_version = 1;
+
+  // All the voiceover corresponds to the LanguageProtosVersion.
+  repeated VoiceoverContentMapping voiceover_language_mapping = 2;
+}
+
+// Structure corresponds to LanguageProtosVersion.
+message VoiceoverContentMapping {
+  LanguageCode language_code = 1;
+  map<string, Voiceover> voiceover_content_mapping = 2;
+}
+
+// Structure corresponds to LanguageProtosVersion.
+message Voiceover {
+
+  // File name of the voiceover.
+  string filename = 1;
+
+  // Size of voiceover file. 
```

Comment:
The size of the voiceover file, in bytes.

Reply:
done